kairon/shared/chat/user_media.py

```
@@ -534,6 +534,16 @@ def get_media_ids(bot: str):
         except Exception as e:
             raise AppException(f"Error while fetching media ids for bot '{bot}': {str(e)}")
 
+    @staticmethod
+    def delete_media(bot, media_id: str):
+        try:
+            UserMediaData.objects(
+                bot=bot,
+                media_id=media_id
+            ).delete()
+            return "Deleted successfully"
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)